ui/redux/actions/content.js

```diff
@@ -220,6 +220,15 @@ export function savePosition(uri: string, position: number) {
     const { claim_id: claimId, txid, nout } = claim;
     const outpoint = `${txid}:${nout}`;
 
+    localStorage.setItem(
```

Comment:
Not sure if still relevant today, but the other localStorage usages will first check if it's available to use.  In the past, some obscure browsers didn't support it.

Reply:
It also seems like it doesn't work in iOS Safari on BrowserStack

Reply:
Ah, right.  While localStorage is pretty widely available, it is not guaranteed to be available, depending on browser settings, such as cookie settings.

I've wrapped it in a try catch to cover this case.